servicebox-app/src/main/java/io/halkyon/ClaimExtendedResource.java

```diff
@@ -39,9 +39,10 @@ public Claim findByName(@PathParam("name") String name) {
 
     @POST
     @Transactional
-    public Response add(@Form Claim claim) {
+    @Consumes("application/x-www-form-urlencoded")
+    public Response add(@Form Claim claim, @HeaderParam("HX-Request") boolean hxRequest) {
         claim.persist();
-        // Return as HTML the template rendering the item
-        return Response.ok(Templates.item(claim)).header("Location", "/claims").build();
+        // Return as HTML the template rendering the item for HTMX
+        return Response.accepted(Templates.item(claim)).status(Response.Status.CREATED).header("Location", "/claims").build();
```

Comment:
We need to return a different response as needed by JSON Test case or if the project do not use HTMX ;-) 
Something like 
```
        if (hxRequest) {
            return Response.ok(Templates.item(claim)).header("Location", "/claims").build();
        }
        return Response.status(Response.Status.CREATED).header("Location", "/claims").build();
```

Reply:
If the project doesn't use HTMX, this method won't be execute. Instead, the method `add`generated will be executed.